juju/models.go

```diff
@@ -162,29 +163,40 @@ func (c *modelsClient) CreateModel(ctx context.Context, input CreateModelInput)
 	client := modelmanager.NewClient(conn)
 	defer client.Close()
 
-	id := fmt.Sprintf("%s/%s/%s", input.Cloud, currentUser, input.CredentialName)
-	if !names.IsValidCloudCredential(id) {
-		return &CreateModelResponse{}, errors.Errorf("%q is not a valid credential id", id)
+	cloudCredTag := names.CloudCredentialTag{}
```

Comment:
if the cloudCredTag is "", we dont want to set the tag as Juju will interpret an empty tag as not requiring creds